### PR TITLE
Add docs label for markdown files

### LINKED ISSUE #656: Treat InternalDocs/* like Docs/*?
I.e. add label "docs" (or similar), "skip news", etc.

An example: https://github.com/python/cpython/pull/127687

### PR DESCRIPTION
**Pull request by skirpichev**
Closes #656

### CODE REVIEW

File: bedevere/prtype.py [hunk 1]
```diff
@@ -43,7 +43,7 @@ async def classify_by_filepaths(gh, pull_request, filenames):
         if util.is_news_dir(filename):
             news = True
         filepath = pathlib.PurePath(filename)
-        if filepath.suffix == ".rst" or filepath.name == ".nitignore":
+        if filepath.suffix in [".rst", ".md"] or filepath.name == ".nitignore":
```
Comment (hugovk) [inline bedevere/prtype.py:46]: Let's put them in alphabetical order:
```suggestion
        if filepath.suffix in [".md", ".rst"] or filepath.name == ".nitignore":
```

**Review by hugovk — APPROVED** [review]
Thanks!

File: bedevere/prtype.py [hunk 2]
```diff
@@ -43,7 +43,7 @@ async def classify_by_filepaths(gh, pull_request, filenames):
         if util.is_news_dir(filename):
             news = True
         filepath = pathlib.PurePath(filename)
-        if filepath.suffix == ".rst" or filepath.name == ".nitignore":
+        if filepath.suffix in [".md", ".rst"] or filepath.name == ".nitignore":
```
Comment (AA-Turner) [inline bedevere/prtype.py:46]: Tiny optimisation (the optimiser might catch this, but stylisticly I'd prefer avoiding membership tests with lists)
```suggestion
        if filepath.suffix in {".md", ".rst"} or filepath.name == ".nitignore":
```

**Review by AA-Turner — APPROVED** [review]

File: tests/test_prtype.py [hunk 3]
```diff
@@ -66,7 +66,7 @@ async def test_news_only():
 
 
 async def test_docs_no_news():
-    filenames = {"path/to/docs1.rst"}
+    filenames = {"path/to/docs1.rst", "other/path/to/docs2.md"}
```
Comment (ezio-melotti) [inline tests/test_prtype.py:69]: Wouldn't this test pass even without the changes in `prtype.py`?
The filenames include an `.rst` file, so the `docs` label should be applied regardless.
I would suggest to verify this, and possibly turn the list of filenames into a fixture.

Comment (skirpichev) [reply tests/test_prtype.py:69]: > Wouldn't this test pass even without the changes in prtype.py?

No.  In this case classify_by_filepaths() will quickly return an empty list, walking over filenames (else branch).

**Pull request merged**